src/pages/HomePage.java

```diff
@@ -0,0 +1,32 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+public class HomePage extends BasePage{
+
+    BasePage page = new BasePage(driver);
+    private static String BASE_URL = "https://testkwidos.tk";
+
+    //locators
+    @FindBy(css = "ul.user.nav li")
+    WebElement signInButton;
```

Comment:
Variables should be private.

Reply:
If I put it private I won't be able to access it.